tests/std/tests/P0645R10_text_formatting_parsing/test.cpp

```diff
@@ -217,5 +276,13 @@ int main() {
     test_parse_precision<wchar_t>();
     static_assert(test_parse_precision<char>());
     static_assert(test_parse_precision<wchar_t>());
+    test_parse_precision<char>();
+    test_parse_precision<wchar_t>();
+    static_assert(test_parse_precision<char>());
+    static_assert(test_parse_precision<wchar_t>());

```

Comment:
Aren't the above redundant with 275-278?

Reply:
merge issue when I rebased onto the previous PR.